channeldb/channel_test.go

```diff
@@ -1352,7 +1359,7 @@ func TestCloseInitiator(t *testing.T) {
 			// local initiator.
 			updateChannel: func(c *OpenChannel) error {
 				return c.MarkCommitmentBroadcasted(
-					&wire.MsgTx{}, true,
+					&wire.MsgTx{}, lntypes.Local,
```

Comment:
Can you explain more here? afaict this doesn't impact the ability to do a git-bisect since the commit works both prior and after the commit.

Reply:
If we were to introduce a bug/undesired side effect with this commit and we identified it using biset, there's still one big commit that touches many parts of the codebase that we'd need to sift through. If we split it up into smaller logical commits (eg, one commit per subsystem), bisect becomes more precise and it's easier to find what went wrong.

So tl;dr a very roundabout way of suggesting that the commit that introduces `ChannelParty` is broken up into smaller parts 🙏 would also make it easier (at least for me) to review

Reply:
I'll give it a try. It's been a while since I authored this commit. IIRC there were a lot of cascading changes as a result of parameters being threaded through multiple layers of functions. I'll try and break it up by subsystem though and see what I can do here.

Reply:
So one of the commits is still gonna be massive and the work to break it up is gonna be far more costly than it was to do the breakup across files. Hopefully it helps.